docs/topics/time.md

```diff
@@ -0,0 +1,425 @@
+   val mark = timeSource.markNow()
+//sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="kotlin-time-mark-moment"}
+
+### Measure differences in time
+
+To measure differences between `TimeMarks` from the same time source, use the subtraction operator (`-`).
+
+To compare `TimeMarks` from the same time source, use the comparison operators (`<`, `>`).
+
+For example:
+
+```kotlin
+import kotlin.time.*
+fun main() {
+//sampleStart
+   val timeSource = TimeSource.Monotonic
+   val mark1 = timeSource.markNow()
+   Thread.sleep(500) // Sleep 0.5 seconds.
+   val mark2 = timeSource.markNow()
+
+   repeat(4) { n ->
+       val mark3 = timeSource.markNow()
+       val elapsed1 = mark3 - mark1
+       val elapsed2 = mark3 - mark2
+
+       println("Measurement 1.${n + 1}: elapsed1=$elapsed1, elapsed2=$elapsed2, diff=${elapsed1 - elapsed2}")
+   }
+   
+   println(mark2 > mark1) // This is true, as mark2 was captured later than mark1.
+   // true
+//sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="kotlin-time-measure-difference"}
+
+To check if a deadline has passed or a timeout has been reached, use the [`hasPassedNow()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-time-mark/has-passed-now.html)
+and [`hasNotPassedNow()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.time/-time-mark/has-not-passed-now.html) 
+extension functions:
+
+```kotlin
+import kotlin.time.*
+import kotlin.time.Duration.Companion.seconds
+
+fun main() {
+//sampleStart
+   val timeSource = TimeSource.Monotonic
+   val mark1 = timeSource.markNow()
+   val fiveSeconds: Duration = 5.seconds
+   val mark2 = mark1 + fiveSeconds
+
+   // It hasn't been 5 seconds yet
+   println(mark2.hasPassedNow())
+   // false
+  
+   // Wait six seconds
+   Thread.sleep(6000)
+   println(mark2.hasPassedNow())
+   // true
+
+//sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="kotlin-time-deadline=passed"}
+
+## Time sources
+
+By default, time is measured using a monotonic time source. Monotonic time sources only move forward and are not affected
+by variations like timezones. An alternative to monotonic time is elapsed real time which is also known as wall-clock time.
+Elapsed real time is time measured relative to another point in time.
+
+This section contains a list of default monotonic time sources per platform, as well as instructions on how to [create
+a non-monotonic time source](#create-non-monotonic-time-source).
+
+### Monotonic time sources per platform
+
+This table explains the default source of monotonic time for each platform:
+
+| Platform      | Source |
+|---------------|---|
+| Kotlin/JVM    | `System.nanoTime()`|
+| Kotlin/JS (Node.js) | `process.hrtime()`|
+| Kotlin/JS (Browser) | `window.performance.now()` or `Date.now()`|
+| Kotlin/Native | `std::chrono::high_resolution_clock` or `std::chrono::steady_clock`|
+
+### Create non-monotonic time source
```

Comment:
This is a **monotonic** time source, as it can be seen from its name: `RealtimeMonotonicTimeSource`